sentry-sample/build.gradle.kts

```diff
@@ -58,7 +58,8 @@ android {
 dependencies {
     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
 
-    implementation(project(":sentry-android"))
+//    implementation(project(":sentry-android"))
+    implementation("io.sentry:sentry-android:2.0.0-SNAPSHOT")
```

Comment:
Does this mean we need to run `release` task before updating the sample?
Or can we just change code and hit 'play' on IntelliJ for example?

Reply:
@bruno-garcia I've inverted the comment, my bad.